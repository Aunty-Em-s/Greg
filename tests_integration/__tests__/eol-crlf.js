"use strict";

const runPrettier = require("../runPrettier");

expect.addSnapshotSerializer(require("../path-serializer"));

describe("Has correct default Windows line endings", () => {
  runPrettier("cli/config/", ["**/*.js"]).test({
    status: 0
  });
});
