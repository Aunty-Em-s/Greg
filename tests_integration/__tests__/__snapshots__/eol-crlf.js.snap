// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Has correct default Windows line endings (stderr) 1`] = `""`;

exports[`Has correct default Windows line endings (stdout) 1`] = `
"function f() {
	console.log(\\"should have tab width 8\\")
}
function f() {
  console.log(\\"should have space width 2\\")
}
function f() {
        console.log(\\"should have space width 8\\")
}
function f() {
  console.log(
    \\"should have space width 2 despite ../.editorconfig specifying 8, because ./.hg is present\\"
  )
}
console.log(\\"jest/__best-tests__/file.js should have semi\\");/*CR*/
console.log(\\"jest/Component.js should not have semi\\")/*CR*/
console.log(\\"jest/Component.test.js should have semi\\");/*CR*/
function js() {
        console.log(\\"js/file.js should have tab width 8 (1 if CLI)\\");
}
\\"use strict\\";

module.exports = {
        endOfLine: \\"lf\\",
        tabWidth: 8
};
function noConfigJs() {
  console.log(\\"no-config/file.js should have no semicolons\\")
}
function packageJs() {/*CR*/
   console.log(\\"package/file.js should have tab width 3\\");/*CR*/
}/*CR*/
function rcJson() {/*CR*/
  console.log.apply(null, [/*CR*/
    'rc-json/file.js',/*CR*/
    'should have trailing comma',/*CR*/
    'and single quotes',/*CR*/
  ]);/*CR*/
}/*CR*/
function rcToml() {/*CR*/
  console.log.apply(null, [/*CR*/
    'rc-toml/file.js',/*CR*/
    'should have trailing comma',/*CR*/
    'and single quotes',/*CR*/
  ]);/*CR*/
}/*CR*/
function rcYaml() {/*CR*/
  console.log.apply(null, [/*CR*/
    'rc-yaml/file.js',/*CR*/
    'should have trailing comma',/*CR*/
    'and single quotes',/*CR*/
  ]);/*CR*/
}/*CR*/
"
`;

exports[`Has correct default Windows line endings (write) 1`] = `Array []`;
